gothemis/message/message.go

```diff
@@ -49,6 +66,13 @@ import (
 	"unsafe"
 )
 
+const (
+	SECURE_MESSAGE_ENCRYPT = 0
```

Comment:
what about to use `iota` value - https://yourbasic.org/golang/iota/ ?
plus better to use [MixedCapsName](https://golang.org/doc/effective_go.html#mixed-caps) for constants to be convient with golang conventions

Reply:
Makes sense, I've updated the code to use more idiomatic enumeration.